### ISSUE TITLE
Graph not connected false error

### ISSUE THREAD

**Issue by asier-gutierrez** [benedekrozemberczki/karateclub#17]
When using GraphWave method with the same graph works on version 0.45.10 of karateclub but using the last versions shows an error of 'Graph not connected'. In that previous version the resulting embeddings were fine and worked well in tasks such as regression with Tensorflow. So it might be an error with the new version regarding the validation of the graph.

Thanks for this awesome python library!

**Comment by benedekrozemberczki** [comment]
Are you sure your graph is connected?

**Comment by benedekrozemberczki** [comment]
What happens when you run nx.is_connected(graph)?

**Comment by asier-gutierrez** [comment]
I have found the issue, it is related with the graph itself and how previously the connectivity was checked.

The graph contained duplicated nodes with the same IDs (I don't know why). The  nx.is_connected() function function spots incoherences like that one and thus avoids the embeddings from being computed.

TLDR; it works

**Comment by benedekrozemberczki** [comment]
Cool :)
